app/src/main/java/uk/gov/onelogin/features/AppCheckFeatureFlag.kt

```diff
@@ -0,0 +1,7 @@
+package uk.gov.onelogin.features
+
+import uk.gov.android.features.FeatureFlag
+
+enum class AppCheckFeatureFlag(override val id: String = "appCheckEnabled") : FeatureFlag {
```

Comment:
If this only has one value, and it looks like it will always have one value, is it not better to use an object for this?

Reply:
I do tend to agree, although I could imagine that we might have multiple entries for the Wallet flags that are coming up. Not too sure why it was build this way, but the `FeatureFlag` interface and style was carried over from ID check